cmake/project_version.cmake

```diff
@@ -0,0 +1,45 @@
+# Set a default version for archive builds that do not have
+# git history.  Increment this appropriately at tag-and-release time.
```

Comment:
If we have to do this anyway is there much benefit to using `git describe`?

I wonder if it's simpler just to update it manually.

Reply:
I think `git describe` would be useful for users who don't wait for released versions, e.g. the ACT folks (see #850).  I think they also originated the discussion in #109.